index.html

```diff
@@ -423,7 +423,9 @@
         </paper-toolbar>
 
         <iron-pages id="pages" selected="[[tab]]" attr-for-selected='id'>
-          <default-page id="default" edit="{{edit}}" route="{{route}}"></default-page>
+          <template is="dom-if" id="default-template" if="[[showPage('default', tab)]]">
```

Comment:
I also wrapped `default-page` in the same `dom-if` template pattern so it isn't loaded on every page load.